src/averageAges.js

```diff
@@ -15,11 +16,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = century
+    ? people.filter((p) => p.sex === 'm' && Math.ceil(p.died / 100) === century)
+    : people.filter((p) => p.sex === 'm');
```

Comment:
Write a better name for the array method's 'el' attribute.

  Consider using a more descriptive name like 'person' instead `p`.

  Example:

```suggestion
  const men = century
    ? people.filter((person) => person.sex === 'm' && Math.ceil(person.died / 100) === century)
    : people.filter((person) => person.sex === 'm');
```


it is also better to move the ternary operator inside the filtering methods instead of writing two filtering methods

```suggestion
   const men = people.filter(person => {
    return century
      ? person.sex === 'm' && Math.ceil(person.died / 100) === century
      : person.sex === 'm';
  });
```